### DISCUSSION TITLE
Lemon Squeezy support

### DISCUSSION THREAD

**Discussion by excid3** [pay-rails/pay#897] [Ideas]
I think we should add support for Lemon Squeezy. The interface is pretty similar to Paddle and would be a nice addition to the library that I don't think we'd mind maintaining since it shouldn't be as complicated as some of the others.

@deanpcmad - looks like the lemonsqueezy gem is a slightly behind the API. https://docs.lemonsqueezy.com/api/changelog

How much work would it be to finish that up? It looks like it'd take an hour or so to implement the missing endpoints and probably the existing ones haven't changed much?

**Comment by deanpcmad** [comment]
Funnily enough, I did start on a Lemon Squeezy integration a few months ago 😆
I've got a branch on my fork of the work I've done so far, so I can start a PR for that

**Reply by excid3** [reply]
Oh nice! I'm debating on including LemonSqueezy in the Pay v7 release, but not sure. Don't want to continue extending this out too long, but also didn't want to quickly follow up with a Pay v8 release, but I guess LS could be added in a 7.1 release or something.

**Reply by lebedevilya** [reply]
what is the status on lemonsqueezy?

**Reply by deanpcmad** [reply]
@lebedevilya I created PR #947 for what I've done so far. I need to finish it off but feel free to test and comment on that PR
